### DISCUSSION TITLE
Access-Control-Allow-Origin

### DISCUSSION THREAD

**Discussion by RandomSamurai** [linuxserver/Heimdall#1063] [General]
Hi 

I am trying to add some functionality to Heimdall such as the current IP of the internet connection.
But need to enable to Access-Control-Allow-Origin to be able to work.  What would the best way be to enable this?  
I have tried adding it to a few different files, but no success.  Does anyone know the best way to do this?

$.getJSON("https://api.myip.com", function(data) {
  // Display the visitor's IP in the console
  console.log(data.ip);
});

**Comment by keriati** [comment]
In the above example the CORS header would need to be set on myip.com and not on heimdall.
